### PR TITLE
#168263381 Create a fullscreen popup for session rating

### PR DESCRIPTION
**Pull request by higustave12**
#### What does this PR do?
Create a fullscreen popup for session rating and add comment review text box 
#### Description of Task to be completed?
This Pull request Create a fullscreen popup for session rating and adds a text box for the review. Once a mentee clicks on "Review session" button, a fullscreen popup opens. Thus, a mentee can input rate value into the text input box and add a comment in text box 
#### How should this be manually tested?
- Clone this repository to the computer 
- Navigate to FreeMentorsUI/Free-Mentors_ChallengeOne/UI/HTML/mentee_dashboard.html
- Double click at mentee_dashboard.html
- Click on "Review session" button
#### What are the relevant pivotal tracker stories?
[Story link](https://www.pivotaltracker.com/story/show/168263381)
#### Screenshots
image

### CODE REVIEW

**Review by oesukam — CHANGES_REQUESTED** [review]
You also need the text box for the review

**Comment by higustave12** [comment]
Thank you for the review. I now finished to add the review text box.The PR
is available for your review.
Thank you

**Review by oesukam — CHANGES_REQUESTED** [review]
You should also update your screenshots with the last changes

**Comment by higustave12** [comment]
Last changes screenshot updated

File: UI/JS/mentee_dashboard.js [hunk 1]
```diff
@@ -112,5 +128,6 @@ const Clocktime = () => {
             document.getElementById("mentor_review_div").style.display="none";
             document.getElementById("review_session_btn").disabled= "true";
           }
+        }
       }
   }
```
Comment (oesukam) [inline UI/JS/mentee_dashboard.js:134]: Add blank line

**Review by oesukam — CHANGES_REQUESTED** [review]
You should use the text area instead of input for the review

File: UI/JS/mentee_dashboard.js (hunk 1 above)
Comment (higustave12) [reply UI/JS/mentee_dashboard.js:134]: Where do i add blank line?

Comment (oesukam) [reply UI/JS/mentee_dashboard.js:134]: On the line that I commented, it is at the end of file

Comment (higustave12) [reply UI/JS/mentee_dashboard.js:134]: I have added a blank line

**Review by oesukam — APPROVED** [review]

**Pull request merged**